doc/modules/manifold.rst

```diff
@@ -457,6 +460,19 @@ order to avoid that, the disparities :math:`\hat{d}_{ij}` are normalized.
    :align: center
    :scale: 60
 
+.. _multidimensional_scaling_method:
+
+Method
+-------------
+
+Metric :class:`MDS` offers two different algorithms (methods) to calculate
```

Comment:
Maybe this should not be under nonmetric

Reply:
Done